app/labor/sticky_article_collection.rb

```diff
@@ -24,7 +24,7 @@ def tag_articles
     @tag_articles ||= Article.published.tagged_with(article_tags, any: true).
       includes(:user).
       where("positive_reactions_count > ? OR comments_count > ?", reaction_count_num, comment_count_num).
-      where.not(id: article.id, user_id: article.user_id).
+      where.not(id: article.id).where.not(user_id: article.user_id).
```

Comment:
Rails 6.1 is going to change the algorithm from NOR to NAND if you pass multiple arguments to a `where.not`, so I fixed the deprecation, see https://github.com/rails/rails/blob/6-0-stable/activerecord/CHANGELOG.md#rails-600rc1-april-24-2019